### PR TITLE
Update index.md with logout URL example

### PR DESCRIPTION
**Pull request by maikell**
Logout URL should also be defined to make a logout request possible. 

 The image can also maybe be updated with: https://2914113074-files.gitbook.io/~/files/v0/b/gitbook-x-prod.appspot.com/o/spaces%2FUeb1lPrWdn7TlqnXoAAC%2Fuploads%2FI3XW6LRpMd4Y1XsxPGqn%2F2.15-settings-authentication-oauth-custom.png?alt=media&token=e5eaa042-9c83-492c-8381-0d957a3c7b2e

### CODE REVIEW

File: docs/content/en/integration/openid-connect/portainer/index.md [hunk 1]
```diff
@@ -60,8 +60,9 @@ To configure [Portainer] to utilize Authelia as an [OpenID Connect 1.0] Provider
    7. Access Token URL: `https://auth.example.com/api/oidc/token`
    8. Resource URL: `https://auth.example.com/api/oidc/userinfo`
    9. Redirect URL: `https://portainer.example.com`
-   10. User Identifier: `preferred_username`
-   11. Scopes: `openid profile groups email`
+   10. Logout URL:  `https://auth.example.com/logout`
```
Comment (james-d-elliott) [inline docs/content/en/integration/openid-connect/portainer/index.md:63]:
```suggestion
   10. Logout URL (optional):  `https://auth.example.com/logout`
```

**Review by james-d-elliott — APPROVED** [review]

**Comment by james-d-elliott** [comment]
Can you confirm the behavior of this? Are users given an option in portainer to  also logout of their provider? Does logout of portainer still allow access to portainer? If not what advantage does this have?

**Comment by maikell** [comment]
> Can you confirm the behavior of this? Are users given an option in portainer to also logout of their provider? Does logout of portainer still allow access to portainer? If not what advantage does this have?

When pressed on logout, it will  logout at Authelia, thus Portainer and all other services connected with Authelia.

**Comment by james-d-elliott** [comment]
What occurs if you logout with this method, and then manually login to the authelia portal and then manually visit portainer? Does it require you to consent again or does it just let you login?

**Pull request closed without merging**